### PR TITLE
Refresh images in Scout dashboard

### PR DESCRIPTION
**Pull request by ChrisChinchilla**
Refreshes several images in the Scout dashboard docs, resizes some others, explains changes in structure of the dashboard page and also sneaks a small fix to a URL in.

### CODE REVIEW

File: scout/dashboard.md [hunk 1]
```diff
@@ -121,7 +120,9 @@ Each entry in the list shows the following details:
 - The versions of the package used by images in the organization.
 - The number of images that use the package.
 
-### Base images
+## Base images
+
+The **Base images** tab shows all base images images are based upon in an organization.
```
Comment (fdiogo) [inline scout/dashboard.md:125]: **nitpick:** "base images images are based upon" sounds like a mouthful 😬 

**suggestion:** 
> The **Base images** tab shows all base images used in an organization.

In my opinion, I could be wrong, the term `base image` is pretty much self-explanatory so we shouldn't have to extend the sentence

**Review by fdiogo — APPROVED** [review]

File: scout/dashboard.md (hunk 1 above)
Comment (ChrisChinchilla) [reply scout/dashboard.md:125]: @fdiogo It does sound like a mouthful, but I am not sure it is completely clear as a concept… I guess it is really.

```suggestion
The **Base images** tab shows all base images images used by repositories in an organization.
```

How about that?

Comment (fdiogo) [reply scout/dashboard.md:125]: We still have the double `images` but it is better 👍

Comment (ChrisChinchilla) [reply scout/dashboard.md:125]: Yeah, I know :( Let's use it for now.

**Review by aevesdocker — DISMISSED** [review]

File: scout/dashboard.md [hunk 2]
```diff
@@ -19,8 +19,9 @@ The **Overview** tab shows the total number of vulnerabilities across all your D
 
 ## Repository settings
 
-Enable Docker Scout analysis on repositories in the current organization by selecting your profile
-picture and then the **Repository settings** menu item.
+![A screenshot showing where to find the settings page for Docker Scout](./images/dashboard-overview-settings.png)
```
Comment (usha-mandya) [inline scout/dashboard.md:22]: Do we really need this screenshot here? I believe our users are smart enough to locate the settings icon by reading the description below.

**Comment by usha-mandya** [comment]
We have about 11 screenshots on this page. Considering the maintenance overhead, can we reduce the number of screenshots we have on this page? Also, I notice that for some screenshots, we add a description after the image and there are a couple of screenshots with descriptions before. Could you take a look?

File: scout/dashboard.md (hunk 2 above)
Comment (usha-mandya) [reply scout/dashboard.md]: Docker Desktop has the same icon. It's called the "Settings" icon- https://docs.docker.com/desktop/settings/mac/ , which is generic enough for non-native speakers. Let's use the settings icon to be consistent.

**Review by usha-mandya — APPROVED** [review]

**Pull request merged**